lib/cloudfrontUtil.js

```diff
@@ -214,6 +214,11 @@ function _getPrivateKey(params) {
     privateKeyString = pem.toString('ascii');
   }
 
+  var lineBreakExists = /\r|\n/.exec(privateKeyString);
+  if (!lineBreakExists) {
```

Comment:
Since we're only interested in whether or not the regex found a match, can you use `.test` instead?